src/core/Workspace.ts

```diff
@@ -26,6 +26,9 @@ export default class Workspace {
   }
 
   public static async init(name: string): Promise<Workspace> {
+    if ((await this.exists(name)) === false) {
```

Comment:
Does this need to be explicitly false? Or would this work?
`!(await this.exists(name))`